cognite/neat/rules/models/_rules/dms_architect_rules.py

```diff
@@ -194,23 +198,22 @@ def _create_connection_property(property_: DMSProperty) -> dm.MultiEdgeConnectio
                     if prop.view_property
                 },
             )
-            view_by_ids[view.as_id()] = view
+            view_by_ids[view_id] = view
             if data_model.views is None:
-                data_model.views = [view.as_id()]
+                data_model.views = [view_id]
             else:
-                data_model.views.append(view.as_id())
+                data_model.views.append(view_id)
 
         for dms_view in self.views or []:
             if dms_view.implements:
-                view_id = dm.ViewId(space, dms_view.view, version=version)
+                view_id = dms_view.view.as_id(space, default_version=version)
                 if view_id in view_by_ids:
                     view_by_ids[view_id].implements = [
-                        dm.ViewId(space, parent, version=version) for parent in dms_view.implements
+                        parent.as_id(space, default_version=version) for parent in dms_view.implements
                     ]
-
         return DMSSchema(
-            space=self.metadata.as_space(),
-            model=data_model,
+            spaces=dm.SpaceApplyList([self.metadata.as_space()]),
```

Comment:
Will we have multiple data models defined within the same `Rules` file?

Reply:
No, but just made this more generic so it can handle multiple data models.